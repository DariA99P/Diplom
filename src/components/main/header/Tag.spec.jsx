import React from 'react';
import { shallow } from 'enzyme';

import TagComponent from './Tag';

jest.mock('utils/environmentDefinition', () => () => ({
  code: 'DEV',
  color: 'blue',
  label: 'Dev Environment',
}));

describe('<Tag />', () => {
  describe('On rendering', () => {
    describe('When appliction environment is defferent from PROD', () => {
      const tree = shallow(<TagComponent />);

      it('should renders a tag informing the environment', () => {
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
