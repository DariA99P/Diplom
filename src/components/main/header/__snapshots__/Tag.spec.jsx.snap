// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> On rendering When appliction environment is defferent from PROD should renders a tag informing the environment 1`] = `
<Tag
  closable={false}
  color="blue"
  prefixCls="ant-tag"
  style={
    Object {
      "cursor": "context-menu",
      "marginLeft": "16px",
    }
  }
>
  Dev Environment
</Tag>
`;
